test/integration/mon/serverspec/server_spec.rb

```diff
@@ -2,6 +2,14 @@
 
 set :backend, :exec
 
+describe package('nagios-plugins-http') do
+  it { should be_installed }
+end
+
+describe file('/etc/nagios/nrpe.d/check_http.cfg') do
+  it { should exist }
+end
+
```

Comment:
This was more for me to verify that it is all working as expected. But if you'd prefer removing it I have no strong objections.